packages/oid4vci-demo-frontend/src/ecosystem-config.ts

```diff
@@ -1,61 +1,41 @@
-import {ImageProperties, VCIEcosystem} from "./types";
+import {ImageProperties} from "./types";
 import dbc from "./configs/dbc.json";
 import energy_shr from "./configs/energy_shr.json";
 import fmdm from "./configs/fmdm.json";
+import notary from "./configs/notary.json";
 import sphereon from "./configs/sphereon.json";
 import triall from "./configs/triall.json";
 import {CSSProperties} from "react";
 import {IProps} from "./components/SSISecondaryButton";
 
-export function getCurrentEcosystem(): VCIEcosystem {
-    switch (process.env.REACT_APP_ENVIRONMENT) {
-        case VCIEcosystem.fmdm:
-            return VCIEcosystem.fmdm;
-        case VCIEcosystem.dbc:
-            return VCIEcosystem.dbc;
-        case VCIEcosystem.triall:
-            return VCIEcosystem.triall;
-        case VCIEcosystem.energy_shr:
-            return VCIEcosystem.energy_shr;
-        default:
-            return VCIEcosystem.sphereon;
-    }
-}
-
 interface VCIConfig {
     general: EcosystemGeneralConfig
     pages: VCIConfigPages
     components: VCIConfigComponents
 }
 
+//todo: (VDX-251) come up with a plan to remove this part and resolve the config automatically based on the REACT_APP_ENVIRONMENT value
 export function getCurrentEcosystemConfig(): VCIConfig {
-    switch (getCurrentEcosystem()) {
-        case VCIEcosystem.triall:
-            return triall as VCIConfig;
-        case VCIEcosystem.fmdm:
-            return fmdm as VCIConfig;
-        case VCIEcosystem.dbc:
-            return dbc as VCIConfig;
-        case VCIEcosystem.energy_shr:
-            return energy_shr as VCIConfig;
-        default:
-            return sphereon as VCIConfig;
-    }
+  const ecosystem = process.env.REACT_APP_ENVIRONMENT ?? 'sphereon';
+  switch (ecosystem) {
+    case 'dbc':
```

Comment:
Just do ecosystem as VCIConfig

Reply:
I'm not sure what do you mean by this. ecosystem string is the name of the ecosystem. and for importing dynamically we need it to be asynchronous. and I've created another ticket for that

Reply:
You are litterlay doing the same every time based on the key (ecosystem), Just do `ecosystem as VCIConfig` and be done with it. The only change in behaviour would be that there is no fallback to Sphereon anymore, which there shouldn't be anyway

Reply:
You do not need async. You could just use the filesystem instead of importing it at the top

Reply:
changed the logic here